lizrd/core/init.py

```diff
@@ -0,0 +1,34 @@
+import torch
+from torch.nn.init import trunc_normal_
+
+
+def get_init_weight(shape, fan_in, init_type="uniform", scale=1.0, dtype=torch.float32):
```

Comment:
I don't like the default uniform here. The defaults should be moved to the config files.

Reply:
+1